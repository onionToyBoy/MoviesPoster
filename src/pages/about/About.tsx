import React from 'react';

import styles from './About.module.css';
import zombie from '../../img/zombie1.png'

export const About = () => {
    return (
        <div className={styles.container}>
            <h3 className={styles.title}>About us</h3>
            <div className={styles.content}>
                <div className={styles.text}>
                    Lorem ipsum dolor sit amet consectetur, adipisicing elit. Sequi, ducimus impedit. Velit aut veritatis aspernatur nobis enim a sunt at, officiis quas, veniam unde est ut dolores architecto exercitationem? Libero.
                </div>
                <div className={styles.imageBlock}>
                    <img src={zombie} className={styles.image} alt='Zombie' />
                </div>
                <div className={styles.text}>
                    Lorem ipsum dolor sit amet consectetur, adipisicing elit. Sequi, ducimus impedit. Velit aut veritatis aspernatur nobis enim a sunt at, officiis quas, veniam unde est ut dolores architecto exercitationem? Libero.
                </div>

            </div>
            <div className={styles.contactsBlock}>
                <div>
                    <label className={styles.label}>Phone number:</label>
                    <a href="tel:123-456-7890" className={styles.phone}>123-456-7890</a>
                </div>
                <div>
                    <label className={styles.label}>Email:</label>
                    <a href="webmonsters@gmail.com" target="_blank">webmonsters@gmail.com</a>
                </div>
            </div>
        </div>
    )
}