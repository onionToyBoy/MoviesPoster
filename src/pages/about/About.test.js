import React from 'react';
import { About } from './About';
import { shallow } from 'enzyme';

describe('About test', () => {
	test('Should render About component correct', () => {
		const component = shallow(<About />);
		expect(component).toMatchSnapshot();
	});
});
