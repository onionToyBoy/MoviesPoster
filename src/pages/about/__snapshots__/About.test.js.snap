// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About test Should render About component correct 1`] = `ShallowWrapper {}`;
